hack/make/cross

```diff
@@ -28,6 +29,12 @@ for platform in $DOCKER_CROSSPLATFORMS; do
 			export LDFLAGS_STATIC_DOCKER="" # we just need a simple client for these platforms
 			export BUILDFLAGS=( "${ORIG_BUILDFLAGS[@]/ daemon/}" ) # remove the "daemon" build tag from platforms that aren't supported
 		fi
+		# make windows cross compile static daemon work
+		if [ "${platform}" = "windows/amd64" ]; then
+			export CC=x86_64-w64-mingw32-gcc
+			export CGO_ENABLED=1
+			export LDFLAGS_STATIC_DOCKER="${LDFLAGS_STATIC_DOCKER/-linkmode external/} -extld=${CC}"
+		fi
```

Comment:
I think it'd be better if we move this block into `hack/make/binary` so that even if we're cross-building without `hack/make/cross` things work OK, something like:
```bash
if [ "$(go env GOOS)/$(go env GOARCH)" != "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ]; then
    # must be cross-compiling!
    case "$(go env GOOS)/$(go env GOARCH)" in
        windows/amd64)
            export ....
            ;;
    esac
fi
```